src/ruuvitag_ble/df5_decoder.py

```diff
@@ -42,30 +42,49 @@ def acceleration_vector_mg(self) -> tuple[int, int, int] | tuple[None, None, Non
         az = self.data[6]
         if ax == -32768 or ay == -32768 or az == -32768:
             return (None, None, None)
-
         return (ax, ay, az)
 
     @property
-    def acceleration_total_mg(self) -> float | None:
+    def acceleration_x_mss(self) -> float | None:
+        ax = self.data[4]
+        if ax == -32768:
+            return None
+        return round(ax / 1000.0 * 9.8, 2)
+
+    @property
+    def acceleration_y_mss(self) -> float | None:
+        ay = self.data[5]
+        if ay == -32768:
+            return None
+        return round(ay / 1000.0 * 9.8, 2)
+
+    @property
+    def acceleration_z_mss(self) -> float | None:
+        az = self.data[6]
+        if az == -32768:
+            return None
+        return round(az / 1000.0 * 9.8, 2)
+
+    @property
+    def acceleration_total_mss(self) -> float | None:
         ax, ay, az = self.acceleration_vector_mg
         if ax is None or ay is None or az is None:
             return None
-        return math.sqrt(ax * ax + ay * ay + az * az)
+        # Conversion to m/s^2
+        return round(math.sqrt(ax * ax + ay * ay + az * az) / 1000.0 * 9.8, 2)
```

Comment:
Let's use [`math.hypot`](https://docs.python.org/3/library/math.html#math.hypot) here to let CPython do more of the work.